### ISSUE TITLE
crawl responses have a lot of leavers

### ISSUE THREAD

**Issue by yuyuxu** [kevincon/dotaml#5]
Hi Kevin,

When you are crawling, have you encountered a lot of leavers (7 out of 10 games), also a lot of "503 http service not available" ? If so, what did you do to overcome them ?

Thanks!

**Comment by kevincon** [comment]
I don't think we had any of the 503 errors, but I think we did see a lot of leavers. Keep in mind this project was done over 2 years ago and thus the underlying libraries/APIs the data collector uses might not work the same anymore, if at all.
